media/css/firefox/whatsnew/includes/_base.scss

```diff
@@ -0,0 +1,8 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import 'header';
+@import 'footer';
+@import 'release-notes';
+@import 'dark-mode';
```

Comment:
I'm on the fence about including this in the base and activating it with the `t-dark-mode` class in the template. 

Maybe it's better to leave out of the base and optionally import separately to the page's stylesheet? The would remove the need for a dark mode theme class in the template.

Reply:
I think this makes sense? 👍  

It could also be confusing to have both `t-dark` and `t-dark-mode` throughout various templates, when they do very different things?

Reply:
I think moving this to an optional import is less bug-prone. Will update.